validator/client/runner.go

```diff
@@ -56,7 +57,11 @@ func run(ctx context.Context, v iface.Validator) {
 	sub := km.SubscribeAccountChanges(accountsChangedChan)
 	// Set properties on the beacon node like the fee recipient for validators that are being used & active.
 	if err := v.PushProposerSettings(ctx, km); err != nil {
-		log.Fatalf("Failed to update proposer settings: %v", err) // allow fatal. skipcq
+		if strings.Contains(err.Error(), MevValidatorRegistrationErr) {
```

Comment:
If you follow my other suggestion to make ErrMevValidatorRegistration an error type, then you can use errors.Is. 
```suggestion
		if errors.Is(err, ErrMevValidatorRegistration) {
```